packages/shared/src/components/AlertDot.tsx

```diff
@@ -11,19 +11,33 @@ export enum AlertColor {
   Cabbage = 'bg-theme-color-cabbage',
 }
 
+export enum AlertDotSize {
+  XSmall = 'x-small',
+  Small = 'small',
+}
+
 interface AlertDotProps {
   color: AlertColor;
   className?: string;
   style?: CSSProperties;
+  size?: AlertDotSize;
 }
 
-export const AlertDot = forwardRef<HTMLElement, AlertDotProps>(
-  function AlertDot(
-    { color, className, ...props }: AlertDotProps,
-    ref,
-  ): ReactElement {
-    return (
-      <Dot ref={ref} {...props} className={classNames(color, className)} />
-    );
-  },
-);
+const alertSizeMap: Record<AlertDotSize, string> = {
+  [AlertDotSize.XSmall]: 'w-2 h-2',
+  [AlertDotSize.Small]: 'w-2.5 h-2.5',
+};
```

Comment:
Might be worthwhile checking with design if it's intended there are 2 different sizes or a design glitch.